server/services/Keys.js

```diff
@@ -61,6 +59,20 @@ async function isAPIKeyPresent(apiKey) {
   }
 }
 
+/**
+ * Fetches user by API key
+ * @param {string} apiKey of user
+**/
+async function fetchUserByApiKey(apiKey){
+  try{
+    const key = await Keys.findOne({"key":apiKey});
```

Comment:
Do we need the double quotes on the key. Some place you have used the double quotes some place you haven't in the `findOne` functions. Please clarify.

Reply:
done